core/src/apps/cardano/auxiliary_data.py

```diff
@@ -52,10 +51,7 @@ def validate_auxiliary_data(
     if auxiliary_data.catalyst_registration_parameters:
         fields_provided += 1
         _validate_catalyst_registration_parameters(
-            keychain,
-            auxiliary_data.catalyst_registration_parameters,
-            protocol_magic,
-            network_id,
+            auxiliary_data.catalyst_registration_parameters
```

Comment:
I think the original logic ensured that the protocol magic/network id is consistent with the overall transaction network id/protocol magic - is this check still preserved?

Reply:
I see - address parameters don't contain the network id/protocol magic, the top-level protocol magic/network id is automatically used when deriving from address params, so we don't really need to ensure this consistency here